packages/client/src/objectSet/ObjectSet.test.ts

```diff
@@ -375,22 +377,11 @@ describe("ObjectSet", () => {
     });
   });
 
-  describe("strictNonNull: \"throw\"", () => {
-    it("throws when getting bad data", () => {
-      expect(() =>
-        client(Employee).fetchPage({
-          $__EXPERIMENTAL_strictNonNull: "throw",
-        })
-      ).rejects.toThrowErrorMatchingInlineSnapshot(
-        `[Error: Unable to safely convert objects as some non nullable properties are null]`,
-      );
-    });
-  });
-
   describe.each(["fetchOne", "fetchOneWithErrors"] as const)("%s", (k) => {
     describe("strictNonNull: false", () => {
       describe("includeRid: true", () => {
         it("returns bad data", async () => {
+          // Look at this
```

Comment:
Yea so technically realized, our types are still permissive to having $ __EXPERIMENTAL_strictNonNull being passed in, do we want to restrict that?

Reply:
I am thinking we remove it for now. Its not like we want it showing up for customers right